packages/ui/src/assets/visual/meld.ts

```diff
@@ -0,0 +1,31 @@
+import { svg } from 'lit'
+export const meldSvg = svg`<svg xmlns="http://www.w3.org/2000/svg" width="40" height="42" viewBox="0 0 40 42" fill="none">
```

Comment:
We generally remove the `xmlns="http://www.w3.org` property from the SVGs to optimize the files. Could we remove it?

```suggestion
export const meldSvg = svg`<svg width="40" height="42" viewBox="0 0 40 42" fill="none">
```